src/Payments.Emails/EmailService.cs

```diff
@@ -28,27 +29,29 @@ public interface IEmailService
 
     public class SparkpostEmailService : IEmailService
     {
-        private readonly SparkpostSettings _emailSettings;
+        private readonly SparkpostSettings _sparkpostSettings;
 
         private readonly IMjmlServices _mjmlServices;
 
-        private readonly Address FromAddress = new Address("donotreply@payments-mail.ucdavis.edu", "UC Davis Payments");
+        private readonly SmtpClient _client;
+
+        private readonly MailAddress _fromAddress = new MailAddress("donotreply@payments-mail.ucdavis.edu", "UC Davis Payments");
 #if DEBUG
-        private readonly Address RefundAddress = new Address("jsylvestre@ucdavis.edu", "CAES UC Davis Refunds");
+        private readonly MailAddress _refundAddress = new MailAddress("jsylvestre@ucdavis.edu", "CAES UC Davis Refunds");
 #else
-        private readonly Address RefundAddress = new Address("refunds@caes.ucdavis.edu", "CAES UC Davis Refunds");
+        private readonly MailAddress _refundAddress = new MailAddress("refunds@caes.ucdavis.edu", "CAES UC Davis Refunds");
```

Comment:
@sprucely Might be a good idea to set this to a config value so we can test on peaks-test

Reply:
@jSylvestre RefundAddress is now configurable. You'll need to add `Sparkpost:RefundAddress` to your `secrets.json` if you want to test it. I'll double-check azure portal to make sure it's set...